### PR TITLE
Application overview scaffolding

### PR DESCRIPTION
**Pull request by PavelJankoski**
#### Summary

Implement application overview scaffolding.

#### Changes

- New look of the application overview.

##### Steps

1. Login to console.
2. Navigate to applications.
3. Select application.
4. New application overview should be present.

##### Results

image

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Testing: The steps/process to test this feature are clearly explained including testing for regressions.
- [ ] Infrastructure: If infrastructural changes (e.g., new RPC, configuration) are needed, a separate issue is created in the infrastructural repositories.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/containers/application-title-section/index.js [hunk 1]
```diff
@@ -52,9 +52,9 @@ import { selectApiKeysTotalCount } from '@console/store/selectors/api-keys'
 
 const m = defineMessages({
   lastSeenAvailableTooltip:
-    'The elapsed time since the network registered activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.',
+    'The elapsed time since the network registered activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.(old)',
```
Comment (ryaplots) [inline pkg/webui/console/containers/application-title-section/index.js]: What's the reason for this change? If these messages are outdated shouldn't we update them or remove?

**Review by ryaplots — CHANGES_REQUESTED** [review]
LGTM! Just a small comment, there is one thing I don't understand.

File: pkg/webui/console/containers/application-title-section/index.js (hunk 1 above)
Comment (kschiffer) [reply pkg/webui/console/containers/application-title-section/index.js]: I guess this is a workaround to avoid having duplicate message definitions since this whole component will be deleted anyway.

Comment (PavelJankoski) [reply pkg/webui/console/containers/application-title-section/index.js]: Yes correct. Do we delete the components that will not be used anymore right away? @kschiffer @ryaplots

Comment (kschiffer) [reply pkg/webui/console/containers/application-title-section/index.js]: Yeah, I think we should.

File: pkg/webui/console/containers/application-overview-header/application-overview-header.styl [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.root
+  position: sticky
+  top: 0
+  z-index: "calc(%s - 1)" % ($zi.dropdown)
```
Comment (kschiffer) [inline pkg/webui/console/containers/application-overview-header/application-overview-header.styl]: Weird calculation, can't we do this with the regular z indices? And if not couldn't we add another index?

Also just `$zi.dropdown - 1` should work, no?

**Review by kschiffer — APPROVED** [review]

**Review by ryaplots — APPROVED** [review]
